justfile

```
@@ -78,6 +80,13 @@ run-regtest args="":
     --dart-define="ESPLORA_ENDPOINT={{public_regtest_esplora}}" --dart-define="COORDINATOR_P2P_ENDPOINT={{public_regtest_coordinator}}" \
     --dart-define="COORDINATOR_PORT_HTTP={{public_coordinator_http_port}}"
 
+run-local args="":
+    #!/usr/bin/env bash
+    cd mobile && flutter run {{args}} --dart-define="COMMIT=$(git rev-parse HEAD)" --dart-define="BRANCH=$(git rev-parse --abbrev-ref HEAD)" \
+    --dart-define="ESPLORA_ENDPOINT=http://{{local_ip}}:3000" --dart-define="COORDINATOR_P2P_ENDPOINT=02dd6abec97f9a748bf76ad502b004ce05d1b2d1f43a9e76bd7d85e767ffb022c9@{{local_ip}}:9045" \
+    --dart-define="COORDINATOR_PORT_HTTP=8000" --flavor local
```

we could use cURL to quickly fetch the local ip (it's safe to assume we have it on our dev machines.)

```suggestion
    $LOCAL_IP=(curl -sS ifconfig.co)
    cd mobile && flutter run {{args}} --dart-define="COMMIT=$(git rev-parse HEAD)" --dart-define="BRANCH=$(git rev-parse --abbrev-ref HEAD)" \
    --dart-define="ESPLORA_ENDPOINT=http://${LOCAL_IP}:3000" --dart-define="COORDINATOR_P2P_ENDPOINT=02dd6abec97f9a748bf76ad502b004ce05d1b2d1f43a9e76bd7d85e767ffb022c9@{{local_ip}}:9045" \
    --dart-define="COORDINATOR_PORT_HTTP=8000" --flavor local
```